.gitattributes

```diff
@@ -0,0 +1,10 @@
+.appveyor.yml export-ignore
+.composer export-ignore
+.gitattributes export-ignore
+.github export-ignore
+.gitignore export-ignore
+.php_cs.dist export-ignore
+.travis.yml export-ignore
+build export-ignore
+phpunit export-ignore
+phpunit.xml.dist export-ignore
```

Comment:
`.editorconfig`should probably be added here too. It is useful when contributing to Symfony, but you will do that on a source install anyway to be able to commit your changes.

Reply:
Thank you. 

I've added it.